clusterloader2/README.md

```diff
@@ -85,10 +85,11 @@ export CL2_ACCESS_TOKENS_QPS=5
 ## Measurement
 
 Currently available measurements are:
-- **APIResponsiveness** \
-This measurement creates summary for latency and number for server api calls.
+- **APIResponsivenessPrometheusSimple** \
+This measurement creates summary for latency and number for server api calls based on the data collected by the prometheus-operator. 
```

Comment:
s/prometheus-operator/prometheus server/

Reply:
could you please clarify your comments? do you mean replace prometheus-operator to "prometheus-operator/prometheus server"?

Reply:
yes - sorry for not being clear

Reply:
thanks and just updated.